pages/task/read/routers/read.js

```diff
@@ -38,11 +38,16 @@ module.exports = () => {
     const model = get(req.task, 'data.model');
 
     if (model === 'project') {
+      const versionId = get(req.task, 'data.data.version');
+      const project = req.task.data.modelData;
+
       req.projectId = get(req.task, 'data.id');
-      req.establishmentId = get(req.task, 'data.data.establishmentId');
-      return req.api(`/establishment/${req.establishmentId}/project/${req.projectId}`, { query: { withDeleted: true } })
+      req.establishmentId = project.establishmentId;
+
+      return req.api(`/establishment/${req.establishmentId}/project/${req.projectId}/project-version/${versionId}`)
```

Comment:
Does this need a `withDeleted`?

Reply:
I don't think we handle withDeleted in the api for projectVersions, this used to be project